src/chrome/content/rules/Ethical_Data.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="Ethical Data">
+
+	<target host="ethicaldata.co.uk" />
+	<target host="*.ethicaldata.co.uk" />
```

Comment:
Is there any benefit from this being a wildcard? Are they using numerous and random domains?